eclair-core/src/main/resources/reference.conf

```diff
@@ -294,6 +294,8 @@ eclair {
     // it can still be manually fee-bumped using the bumpforceclose RPC
     // *do not change this unless you know what you are doing*
     spend-anchor-without-htlcs = true
+    // maximum amount of fees we will pay to bump an anchor output when we have no HTLC at risk
+    anchor-without-htlcs-max-fee-satoshis = 10000
```

Comment:
The size of a htlc-less commitment tx is 1110 WU / 278 vB, so that's ~36 sat/vB.